### PR TITLE
Add Celestia v1.6.1

### PR DESCRIPTION
**Pull request by MCOfficer**
kinda refs lukesampson/scoop#3339

### CODE REVIEW

File: bucket/celestia.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+    "description": "A real-time space simulation that lets you experience our universe in three dimensions.",
+    "homepage": "https://celestia.space/",
+    "version": "1.6.1",
+    "license": "GPL-2.0",
```
Comment (Ash258) [inline bucket/celestia.json]: Wrong identifier.
>  Free Software Foundation; either version 2 of the License, or (at your option) any later version.

```suggestion
    "license": "GPL-2.0-or-later",
```

File: bucket/celestia.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    "url": "https://celestia.space/cc/celestia-win-161#celestia-161.exe",
+    "hash": "e7b9945e6aca5f5696d3c3be030963faef4c4991cd184222c7a4afbd72b5fd26",
+    "innosetup": true,
+    "bin": "celestia.exe",
```
Comment (Ash258) [inline bucket/celestia.json]: No shortcuts?

Comment (r15ch13) [reply bucket/celestia.json]: Sorry, clicked merge just now ...
Will fix this :grin:

**Comment by r15ch13** [comment]
Updated with https://github.com/lukesampson/scoop-extras/commit/f5318554a725105ecd12b2d21749f938265f535d

File: bucket/celestia.json (hunk 2 above)
Comment (MCOfficer) [reply bucket/celestia.json]: i am... used to cli stuff. whoops.

**Pull request merged**